src/components/NavBar.astro

```diff
@@ -62,12 +64,19 @@ const { title } = Astro.props;
     }
 
     ul {
-        float: right;
         list-style-type: none;
         overflow: hidden;
         padding: 0;
     }
 
+    div#btnsleft {
+        flex: 1;
+    }
+
+    #logo img {
+        vertical-align: middle;
+    }
```

Comment:
these selectors are overly specific again and should probably use a class instead of an id